graph_nets/demos_tf2/shortest_path.ipynb

```diff
@@ -45,7 +45,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 0,
+   "execution_count": null,
    "metadata": {
     "cellView": "form",
     "colab": {},
```

Comment:
(can'c comment below, but these two should be changed):

`!pip install graph_nets "dm-sonnet<2" "tensorflow_probability<0.9"`
to
`!pip install "graph_nets>=1.1" "dm-sonnet>=2.0.0b0"`

and 
`pip install graph_nets matplotlib scipy "tensorflow>=1.15,<2" "dm-sonnet<2" "tensorflow_probability<0.9"`
to
`pip install "graph_nets>=1.1" matplotlib scipy  "tensorflow>=2.1.0-rc1" "dm-sonnet>=2.0.0b0" tensorflow_probability`